test/integration/user_can_see_if_item_is_retired_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.